assets/xml/overlays/ovl_En_Ganon_Mant.xml

```diff
@@ -0,0 +1,22 @@
+<Root>
+    <File Name="ovl_En_Ganon_Mant" BaseAddress="0x80A23D60" RangeStart="0x1218" RangeEnd="0x4118" Segment="128">
+        <Texture Name="gMantTex" OutName="mant" Format="rgba16" Width="32" Height="64" Offset="0x1218"/>
+
+        <Texture Name="gMantUnusedTex" OutName="mant_unused" Format="rgba16" Width="32" Height="32" Offset="0x2218"/>
+
+        <Array Name="gMant1Vtx" Count="144" Offset="0x2A18">
+            <Vtx/>
+        </Array>
+
+        <DList Name="gMantTexDL" Offset="0x3318"/>
+
+        <!-- ! @bug gMantUnusedTex is 32x32 not 32x64, however this dlist is unused -->
```

Comment:
Is this the right way to document a bug in assets? Or just lack of a better option

Reply:
when we move toward committing the c files it will be easier to just put it right in the file. but for now this is all we got i guess